### ISSUE TITLE
typescript, Draggable.applyBounds type is missing

### ISSUE THREAD

**Issue by miladfm** [greensock/GSAP#361]
There is no type definition for `applyBounds` after create new instance of `Draggable`.
But `Draggable` class has this method.
I use Draggable V3.1.1

```
const draggable = new Draggable(element, { ... });
draggable.applyBounds(newelement);
Error: Property 'applyBounds' does not exist on type 'Draggable'.
```

I use my custom type definition to ignore typescript type error .
`applyBounds(bound: gsap.DOMTarget | Draggable.BoundsMinMax | Draggable.BoundsRectangle | Draggable.BoundsRotation): void;`

Please add it into `draggable.d.ts` by next release

**Comment by OSUblake** [comment]
My bad. I didn't see that method because it's not in the documentation, so the documentation also needs to be updated.
https://greensock.com/docs/v3/Plugins/Draggable

**Comment by miladfm** [comment]
Thanks.

further `isDragging` property is missing too.

`isDragging: boolean`

**Comment by OSUblake** [comment]
Thanks for pointing that out. Both have been added to the next release.

**Comment by ZachSaucier** [comment]
I've added both applyBounds and isDragging to the Draggable docs. May need to hard refresh to see them.

Thanks for the help, @miladfm!

**Comment by jackdoyle** [comment]
Should be fixed in the latest release (3.2.0)
